### DISCUSSION TITLE
how can I uninstall zoxide

### DISCUSSION THREAD

**Discussion by ghost** [ajeetdsouza/zoxide#506] [Q&A] [answered]
I installed zoxide on Ubuntu 20.04 with this command now I want to uninstall. How can I do this?

```
curl -sS https://raw.githubusercontent.com/ajeetdsouza/zoxide/main/install.sh | bash
```

**Comment by ajeetdsouza** [comment, marked as answer]
zoxide is a portable binary, so it's fairly simple to remove:

```sh
rm -i "$(which zoxide)"
```

You might also want to remove manpages:

```sh
rm -i "$HOME/.local/share/man/man1/zoxide"*

**Comment by Rozkalns** [comment]
I had an issues that when executing ``, I got message - 
```shell
$ cd .
zsh: command not found: z
```

then executing
```
$ which cd
cd: aliased to z
```

In order to unalias, simply run 
```shell
$ unalias cd 
```

Then result would be this:
```shell
$ which cd 
cd: shell built-in command
```

**Reply by ajeetdsouza** [reply]
The README specifies adding a line to your shell configuration to setup zoxide, you need to remove that too and restart your shell.

**Reply by maxfenton** [reply]
Note that just doing a `source ~/.zshrc` wasn't enough. ajeetdsouza was right that you need to restart the shell
